Swagger-Jaxrs Module Test Failures

<!---
Thanks for filing an issue 😄 ! Before you submit, please read the following:
 
If you're here to report a security issue, please STOP writing an issue and contact us
at security@swagger.io instead!
 
Search open/closed issues before submitting since someone might have asked the same thing before!
 
Issues on GitHub are only related to problems of Swagger-Core itself. We'll try to offer support here for your use case, but we can't offer help with projects that use Swagger-Core, like Springfox.
 
Likewise, we can't accept features or bugs in the Swagger/OpenAPI specifications themselves, or anything that violates the specifications.
-->
 
<!--- Provide a general summary of the issue in the title above -->
 
| Q                                 | A
| --------------------------------- | -------
| Bug or feature request?           | Bug
| Which Swagger-Core version?       |1.5.18
| Which Java version?               |1.8
| Which JAX-RS framework & version? |RESTEasy 3.0.19 with JAX-RS 2.0
 
<!--- Provide full details about the issue, including any stack trace, errors, actual behaviour, expected behaviour -->
When following the instructions for building from the source (`mvn -N` followed by `mvn install`) there are 2 tests that fail for the swagger-jaxrs module. This is on an unmodified download of the 1.5.18 source from Git. The failed tests are testSwaggerJson and testSwaggerJsonUsingAcceptHeaders. All other modules do not appear to have any issues when building. The testing results I get are as follows:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="3086" started-at="2018-03-07T20:11:37Z" finished-at="2018-03-07T20:11:40Z">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="3086" started-at="2018-03-07T20:11:37Z" finished-at="2018-03-07T20:11:40Z">
      <class name="io.swagger.functional.test.ApiListingResourceIT">
        <test-method status="PASS" signature="checkJetty()[pri:0, instance:io.swagger.functional.test.ApiListingResourceIT@48fa0f47]" name="checkJetty" is-config="true" duration-ms="12" started-at="2018-03-07T15:11:37Z" finished-at="2018-03-07T15:11:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkJetty -->
        <test-method status="FAIL" signature="testSwaggerJson()[pri:0, instance:io.swagger.functional.test.ApiListingResourceIT@48fa0f47]" name="testSwaggerJson" duration-ms="2784" started-at="2018-03-07T15:11:37Z" finished-at="2018-03-07T15:11:40Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [{
  "definitions" : {
    "Car" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Detail" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire_Detail"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Summary" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Car_Summary-or-Sale" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        }
      },
      "type" : "object"
    },
    "Tire" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Tire_Detail" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Widget" : {
      "properties" : {
        "a" : {
          "type" : "string"
        },
        "b" : {
          "type" : "string"
        },
        "id" : {
          "type" : "string"
        }
      },
      "type" : "object"
    }
  },
  "paths" : {
    "/cars/all" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getAll",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return whole car",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return whole car",
        "tags" : [ "cars" ]
      }
    },
    "/cars/detail" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getDetails",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return car detail",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Detail"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car detail",
        "tags" : [ "cars" ]
      }
    },
    "/cars/sale" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getSaleSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Summary-or-Sale"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car sale summary",
        "tags" : [ "cars" ]
      }
    },
    "/cars/summary" : {
      "get" : {
        "consumes" : [ "List" ],
        "description" : "",
        "operationId" : "getSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "$ref" : "#/definitions/Car_Summary"
            }
          }
        },
        "summary" : "Return car summaries",
        "tags" : [ "cars" ]
      }
    },
    "/widgets/{widgetId}" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "Returns a pet when ID <= 10.  ID > 10 or nonintegers will simulate API error conditions",
        "operationId" : "getWidget",
        "parameters" : [ {
          "in" : "path",
          "name" : "widgetId",
          "required" : true,
          "type" : "string"
        } ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Returns widget with matching id"
          }
        },
        "summary" : "Find pet by ID",
        "tags" : [ "widgets" ]
      }
    }
  },
  "swagger" : "2.0",
  "tags" : [ {
    "name" : "cars"
  }, {
    "name" : "widgets"
  } ]
}] but found [{
  "definitions" : {
    "Car" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Detail" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire_Detail"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Summary" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Car_Summary-or-Sale" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        }
      },
      "type" : "object"
    },
    "Tire" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Tire_Detail" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Widget" : {
      "properties" : {
        "a" : {
          "type" : "string"
        },
        "b" : {
          "type" : "string"
        },
        "id" : {
          "type" : "string"
        }
      },
      "type" : "object"
    }
  },
  "paths" : {
    "/cars/all" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getAll",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return whole car",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return whole car",
        "tags" : [ "cars" ]
      }
    },
    "/cars/detail" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getDetails",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return car detail",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Detail"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car detail",
        "tags" : [ "cars" ]
      }
    },
    "/cars/sale" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getSaleSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Summary-or-Sale"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car sale summary",
        "tags" : [ "cars" ]
      }
    },
    "/cars/summary" : {
      "get" : {
        "consumes" : [ "List" ],
        "description" : "",
        "operationId" : "getSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "$ref" : "#/definitions/Car_Summary"
            }
          }
        },
        "summary" : "Return car summaries",
        "tags" : [ "cars" ]
      }
    },
    "/widgets/{widgetId}" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "Returns a pet when ID <= 10.  ID > 10 or nonintegers will simulate API error conditions",
        "operationId" : "getWidget",
        "parameters" : [ {
          "in" : "path",
          "name" : "widgetId",
          "required" : true,
          "type" : "string"
        } ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Returns widget with matching id"
          }
        },
        "summary" : "Find pet by ID",
        "tags" : [ "widgets" ]
      }
    }
  },
  "swagger" : "2.0",
  "tags" : [ {
    "name" : "cars"
  }, {
    "name" : "widgets"
  } ]
}]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [{
  "definitions" : {
    "Car" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Detail" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire_Detail"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Summary" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Car_Summary-or-Sale" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        }
      },
      "type" : "object"
    },
    "Tire" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Tire_Detail" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Widget" : {
      "properties" : {
        "a" : {
          "type" : "string"
        },
        "b" : {
          "type" : "string"
        },
        "id" : {
          "type" : "string"
        }
      },
      "type" : "object"
    }
  },
  "paths" : {
    "/cars/all" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getAll",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return whole car",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return whole car",
        "tags" : [ "cars" ]
      }
    },
    "/cars/detail" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getDetails",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return car detail",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Detail"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car detail",
        "tags" : [ "cars" ]
      }
    },
    "/cars/sale" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getSaleSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Summary-or-Sale"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car sale summary",
        "tags" : [ "cars" ]
      }
    },
    "/cars/summary" : {
      "get" : {
        "consumes" : [ "List" ],
        "description" : "",
        "operationId" : "getSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "$ref" : "#/definitions/Car_Summary"
            }
          }
        },
        "summary" : "Return car summaries",
        "tags" : [ "cars" ]
      }
    },
    "/widgets/{widgetId}" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "Returns a pet when ID <= 10.  ID > 10 or nonintegers will simulate API error conditions",
        "operationId" : "getWidget",
        "parameters" : [ {
          "in" : "path",
          "name" : "widgetId",
          "required" : true,
          "type" : "string"
        } ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Returns widget with matching id"
          }
        },
        "summary" : "Find pet by ID",
        "tags" : [ "widgets" ]
      }
    }
  },
  "swagger" : "2.0",
  "tags" : [ {
    "name" : "cars"
  }, {
    "name" : "widgets"
  } ]
}] but found [{
  "definitions" : {
    "Car" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Detail" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire_Detail"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Summary" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Car_Summary-or-Sale" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        }
      },
      "type" : "object"
    },
    "Tire" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Tire_Detail" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Widget" : {
      "properties" : {
        "a" : {
          "type" : "string"
        },
        "b" : {
          "type" : "string"
        },
        "id" : {
          "type" : "string"
        }
      },
      "type" : "object"
    }
  },
  "paths" : {
    "/cars/all" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getAll",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return whole car",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return whole car",
        "tags" : [ "cars" ]
      }
    },
    "/cars/detail" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getDetails",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return car detail",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Detail"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car detail",
        "tags" : [ "cars" ]
      }
    },
    "/cars/sale" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getSaleSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Summary-or-Sale"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car sale summary",
        "tags" : [ "cars" ]
      }
    },
    "/cars/summary" : {
      "get" : {
        "consumes" : [ "List" ],
        "description" : "",
        "operationId" : "getSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "$ref" : "#/definitions/Car_Summary"
            }
          }
        },
        "summary" : "Return car summaries",
        "tags" : [ "cars" ]
      }
    },
    "/widgets/{widgetId}" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "Returns a pet when ID <= 10.  ID > 10 or nonintegers will simulate API error conditions",
        "operationId" : "getWidget",
        "parameters" : [ {
          "in" : "path",
          "name" : "widgetId",
          "required" : true,
          "type" : "string"
        } ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Returns widget with matching id"
          }
        },
        "summary" : "Find pet by ID",
        "tags" : [ "widgets" ]
      }
    }
  },
  "swagger" : "2.0",
  "tags" : [ {
    "name" : "cars"
  }, {
    "name" : "widgets"
  } ]
}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at io.swagger.functional.test.ApiListingResourceIT.testSwaggerJson(ApiListingResourceIT.java:458)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSwaggerJson -->
        <test-method status="PASS" signature="checkJetty()[pri:0, instance:io.swagger.functional.test.ApiListingResourceIT@48fa0f47]" name="checkJetty" is-config="true" duration-ms="0" started-at="2018-03-07T15:11:40Z" finished-at="2018-03-07T15:11:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkJetty -->
        <test-method status="FAIL" signature="testSwaggerJsonUsingAcceptHeader()[pri:0, instance:io.swagger.functional.test.ApiListingResourceIT@48fa0f47]" name="testSwaggerJsonUsingAcceptHeader" duration-ms="29" started-at="2018-03-07T15:11:40Z" finished-at="2018-03-07T15:11:40Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [{
  "definitions" : {
    "Car" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Detail" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire_Detail"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Summary" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Car_Summary-or-Sale" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        }
      },
      "type" : "object"
    },
    "Tire" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Tire_Detail" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Widget" : {
      "properties" : {
        "a" : {
          "type" : "string"
        },
        "b" : {
          "type" : "string"
        },
        "id" : {
          "type" : "string"
        }
      },
      "type" : "object"
    }
  },
  "paths" : {
    "/cars/all" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getAll",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return whole car",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return whole car",
        "tags" : [ "cars" ]
      }
    },
    "/cars/detail" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getDetails",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return car detail",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Detail"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car detail",
        "tags" : [ "cars" ]
      }
    },
    "/cars/sale" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getSaleSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Summary-or-Sale"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car sale summary",
        "tags" : [ "cars" ]
      }
    },
    "/cars/summary" : {
      "get" : {
        "consumes" : [ "List" ],
        "description" : "",
        "operationId" : "getSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "$ref" : "#/definitions/Car_Summary"
            }
          }
        },
        "summary" : "Return car summaries",
        "tags" : [ "cars" ]
      }
    },
    "/widgets/{widgetId}" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "Returns a pet when ID <= 10.  ID > 10 or nonintegers will simulate API error conditions",
        "operationId" : "getWidget",
        "parameters" : [ {
          "in" : "path",
          "name" : "widgetId",
          "required" : true,
          "type" : "string"
        } ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Returns widget with matching id"
          }
        },
        "summary" : "Find pet by ID",
        "tags" : [ "widgets" ]
      }
    }
  },
  "swagger" : "2.0",
  "tags" : [ {
    "name" : "cars"
  }, {
    "name" : "widgets"
  } ]
}] but found [{
  "definitions" : {
    "Car" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Detail" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire_Detail"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Summary" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Car_Summary-or-Sale" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        }
      },
      "type" : "object"
    },
    "Tire" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Tire_Detail" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Widget" : {
      "properties" : {
        "a" : {
          "type" : "string"
        },
        "b" : {
          "type" : "string"
        },
        "id" : {
          "type" : "string"
        }
      },
      "type" : "object"
    }
  },
  "paths" : {
    "/cars/all" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getAll",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return whole car",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return whole car",
        "tags" : [ "cars" ]
      }
    },
    "/cars/detail" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getDetails",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return car detail",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Detail"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car detail",
        "tags" : [ "cars" ]
      }
    },
    "/cars/sale" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getSaleSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Summary-or-Sale"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car sale summary",
        "tags" : [ "cars" ]
      }
    },
    "/cars/summary" : {
      "get" : {
        "consumes" : [ "List" ],
        "description" : "",
        "operationId" : "getSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "$ref" : "#/definitions/Car_Summary"
            }
          }
        },
        "summary" : "Return car summaries",
        "tags" : [ "cars" ]
      }
    },
    "/widgets/{widgetId}" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "Returns a pet when ID <= 10.  ID > 10 or nonintegers will simulate API error conditions",
        "operationId" : "getWidget",
        "parameters" : [ {
          "in" : "path",
          "name" : "widgetId",
          "required" : true,
          "type" : "string"
        } ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Returns widget with matching id"
          }
        },
        "summary" : "Find pet by ID",
        "tags" : [ "widgets" ]
      }
    }
  },
  "swagger" : "2.0",
  "tags" : [ {
    "name" : "cars"
  }, {
    "name" : "widgets"
  } ]
}]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [{
  "definitions" : {
    "Car" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Detail" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire_Detail"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Summary" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Car_Summary-or-Sale" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        }
      },
      "type" : "object"
    },
    "Tire" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Tire_Detail" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Widget" : {
      "properties" : {
        "a" : {
          "type" : "string"
        },
        "b" : {
          "type" : "string"
        },
        "id" : {
          "type" : "string"
        }
      },
      "type" : "object"
    }
  },
  "paths" : {
    "/cars/all" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getAll",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return whole car",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return whole car",
        "tags" : [ "cars" ]
      }
    },
    "/cars/detail" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getDetails",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return car detail",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Detail"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car detail",
        "tags" : [ "cars" ]
      }
    },
    "/cars/sale" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getSaleSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Summary-or-Sale"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car sale summary",
        "tags" : [ "cars" ]
      }
    },
    "/cars/summary" : {
      "get" : {
        "consumes" : [ "List" ],
        "description" : "",
        "operationId" : "getSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "$ref" : "#/definitions/Car_Summary"
            }
          }
        },
        "summary" : "Return car summaries",
        "tags" : [ "cars" ]
      }
    },
    "/widgets/{widgetId}" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "Returns a pet when ID <= 10.  ID > 10 or nonintegers will simulate API error conditions",
        "operationId" : "getWidget",
        "parameters" : [ {
          "in" : "path",
          "name" : "widgetId",
          "required" : true,
          "type" : "string"
        } ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Returns widget with matching id"
          }
        },
        "summary" : "Find pet by ID",
        "tags" : [ "widgets" ]
      }
    }
  },
  "swagger" : "2.0",
  "tags" : [ {
    "name" : "cars"
  }, {
    "name" : "widgets"
  } ]
}] but found [{
  "definitions" : {
    "Car" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Detail" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "tires" : {
          "items" : {
            "$ref" : "#/definitions/Tire_Detail"
          },
          "type" : "array"
        }
      },
      "type" : "object"
    },
    "Car_Summary" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Car_Summary-or-Sale" : {
      "properties" : {
        "color" : {
          "type" : "string"
        },
        "manufacture" : {
          "type" : "string"
        },
        "model" : {
          "type" : "string"
        },
        "price" : {
          "format" : "int32",
          "type" : "integer"
        }
      },
      "type" : "object"
    },
    "Tire" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Tire_Detail" : {
      "properties" : {
        "brand" : {
          "type" : "string"
        },
        "condition" : {
          "type" : "string"
        }
      },
      "type" : "object"
    },
    "Widget" : {
      "properties" : {
        "a" : {
          "type" : "string"
        },
        "b" : {
          "type" : "string"
        },
        "id" : {
          "type" : "string"
        }
      },
      "type" : "object"
    }
  },
  "paths" : {
    "/cars/all" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getAll",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return whole car",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return whole car",
        "tags" : [ "cars" ]
      }
    },
    "/cars/detail" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getDetails",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Return car detail",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Detail"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car detail",
        "tags" : [ "cars" ]
      }
    },
    "/cars/sale" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "",
        "operationId" : "getSaleSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "items" : {
                "$ref" : "#/definitions/Car_Summary-or-Sale"
              },
              "type" : "array"
            }
          }
        },
        "summary" : "Return car sale summary",
        "tags" : [ "cars" ]
      }
    },
    "/cars/summary" : {
      "get" : {
        "consumes" : [ "List" ],
        "description" : "",
        "operationId" : "getSummaries",
        "parameters" : [ ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "$ref" : "#/definitions/Car_Summary"
            }
          }
        },
        "summary" : "Return car summaries",
        "tags" : [ "cars" ]
      }
    },
    "/widgets/{widgetId}" : {
      "get" : {
        "consumes" : [ "application/json" ],
        "description" : "Returns a pet when ID <= 10.  ID > 10 or nonintegers will simulate API error conditions",
        "operationId" : "getWidget",
        "parameters" : [ {
          "in" : "path",
          "name" : "widgetId",
          "required" : true,
          "type" : "string"
        } ],
        "produces" : [ "application/json" ],
        "responses" : {
          "200" : {
            "description" : "Returns widget with matching id"
          }
        },
        "summary" : "Find pet by ID",
        "tags" : [ "widgets" ]
      }
    }
  },
  "swagger" : "2.0",
  "tags" : [ {
    "name" : "cars"
  }, {
    "name" : "widgets"
  } ]
}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at io.swagger.functional.test.ApiListingResourceIT.testSwaggerJsonUsingAcceptHeader(ApiListingResourceIT.java:476)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:132)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeSingleClass(TestNGDirectoryTestSuite.java:112)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:147)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSwaggerJsonUsingAcceptHeader -->
        <test-method status="PASS" signature="checkJetty()[pri:0, instance:io.swagger.functional.test.ApiListingResourceIT@48fa0f47]" name="checkJetty" is-config="true" duration-ms="0" started-at="2018-03-07T15:11:40Z" finished-at="2018-03-07T15:11:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkJetty -->
        <test-method status="PASS" signature="testSwaggerYaml()[pri:0, instance:io.swagger.functional.test.ApiListingResourceIT@48fa0f47]" name="testSwaggerYaml" duration-ms="158" started-at="2018-03-07T15:11:40Z" finished-at="2018-03-07T15:11:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSwaggerYaml -->
        <test-method status="PASS" signature="checkJetty()[pri:0, instance:io.swagger.functional.test.ApiListingResourceIT@48fa0f47]" name="checkJetty" is-config="true" duration-ms="0" started-at="2018-03-07T15:11:40Z" finished-at="2018-03-07T15:11:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkJetty -->
        <test-method status="PASS" signature="testSwaggerYamlUsingAcceptHeader()[pri:0, instance:io.swagger.functional.test.ApiListingResourceIT@48fa0f47]" name="testSwaggerYamlUsingAcceptHeader" duration-ms="41" started-at="2018-03-07T15:11:40Z" finished-at="2018-03-07T15:11:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSwaggerYamlUsingAcceptHeader -->
      </class> <!-- io.swagger.functional.test.ApiListingResourceIT -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>

```

I tried using the swagger-jaxrs jar generated using `mvn package` but I ended up with a runtime error at line 845 of Reader.java:
```
Caused by: java.lang.NoSuchMethodError: io.swagger.annotations.ApiOperation.ignoreJsonView()Z
	at io.swagger.jaxrs.Reader.parseMethod(Reader.java:845)
	at io.swagger.jaxrs.Reader.read(Reader.java:323)
	at io.swagger.jaxrs.Reader.read(Reader.java:173)
	at io.swagger.jaxrs.config.BeanConfig.scanAndRead(BeanConfig.java:242)
	at io.swagger.jaxrs.config.BeanConfig.setScan(BeanConfig.java:221)
	at com.trendmicro.ds.api.ApiApplication.SetBeanConfig(ApiApplication.java:241)
	at com.trendmicro.ds.api.ApiApplication.<init>(ApiApplication.java:99)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.jboss.resteasy.core.ConstructorInjectorImpl.construct(ConstructorInjectorImpl.java:150)
```

--- a/modules/swagger-jaxrs/src/test/java/io/swagger/functional/test/ApiListingResourceIT.java
+++ b/modules/swagger-jaxrs/src/test/java/io/swagger/functional/test/ApiListingResourceIT.java
@@ -455,7 +455,7 @@ public class ApiListingResourceIT {
 
         System.out.println(EXPECTED_JSON);
         System.out.println(formatJson(actualBody));
-        assertEquals(formatJson(actualBody), EXPECTED_JSON);
+        assertEquals(normalizeLineEnds(formatJson(actualBody)), EXPECTED_JSON);
     }
 
     @Test
@@ -473,7 +473,7 @@ public class ApiListingResourceIT {
                 .contentType(ContentType.JSON)
                 .extract().response().body().asString();
 
-        assertEquals(formatJson(actualBody), EXPECTED_JSON);
+        assertEquals(normalizeLineEnds(formatJson(actualBody)), EXPECTED_JSON);
     }
 
     @Test
@@ -521,5 +521,9 @@ public class ApiListingResourceIT {
         return Json.mapper().configure(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS, true)
                 .writerWithDefaultPrettyPrinter()
                 .writeValueAsString(Json.mapper().readValue(source, Object.class));
+    }
+
+    private static String normalizeLineEnds(String s) {
+        return s.replace("\r\n", "\n").replace('\r', '\n');
     }
 }

